js/LICENSE

```diff
@@ -0,0 +1,238 @@
+                                 Apache License
```

Comment:
Remove this? We can bundle the license from `../LICENSE` if needed in the NPM package